cpanfile

```diff
@@ -48,6 +48,7 @@ requires 'Module::Runtime';
 requires 'Mojo::Base';
 requires 'Mojo::ByteStream';
 requires 'Mojo::IOLoop';
+requires 'Mojo::IOLoop::ReadWriteProcess';
```

Comment:
created request id 575491

Reply:
@mudler failed https://build.opensuse.org/package/live_build_log/openSUSE:Factory:Staging:adi:42/perl-Mojo-IOLoop-ReadWriteProcess/standard/x86_64

Reply:
Looking into it

Reply:
0.11 released, in 0.10 the building module changed, but missed updating it in the cpanfile , https://build.opensuse.org/package/live_build_log/home:EDiGiacinto/perl-Mojo-IOLoop-ReadWriteProcess/openSUSE_Tumbleweed/i586